lighthouse-core/gather/driver.js

```diff
@@ -675,20 +697,26 @@ class Driver {
     });
   }
 
-  endNetworkCollect() {
+  endNetworkCollect(opts = {}) {
+    const pauseAfterLoadMs = (opts.flags && opts.flags.pauseAfterLoad) || PAUSE_AFTER_LOAD;
```

Comment:
also as an aside, the implementation of consistently interactive (#2023) increases the pauseafterload to 5s which would be kinda painful to wait an extra 5s here too